java-development/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM openjdk:8u252-slim-buster
```

Comment:
Please use the same ordering as  https://github.com/eventuate-tram/eventuate-tram-examples-customers-and-orders/commit/ca1e1b4299da4c957c1e68f3f4ee4eebf088d94f - commonalities near the top, differences at the end.

Reply:
ok